src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_strategy_selection_test.py

```diff
@@ -58,4 +62,84 @@ class TestStrategySelectionPatchless(unittest.TestCase):
   def test_strategy_pool_generator(self):
     """Ensures that a call to generate_strategy_pool does not yield an
```

Comment:
nit: comment seems to be out of date, now it tests `generate_random_strategy_pool` or whatever we may rename it to.